<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2400</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>279</width>
      <height>373</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="circkeRadio">
       <property name="text">
        <string>Круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="triangleRadio">
       <property name="text">
        <string>Труегольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="squareRadio">
       <property name="text">
        <string>Квадрат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rhombRadio">
       <property name="text">
        <string>Ромб</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rectangleRadio">
       <property name="text">
        <string>Прямоугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="pentagonRadio">
       <property name="text">
        <string>Пятиугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HexRadio">
       <property name="text">
        <string>Шестиугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="star5Radio">
       <property name="text">
        <string>Звезда 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="star6Radoi">
       <property name="text">
        <string>Звезда 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="star8Radoi">
       <property name="text">
        <string>Звезда 8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hendRadio">
       <property name="text">
        <string>Рука</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="penRadoi">
       <property name="text">
        <string>Кисть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="moveTrianglePointsRadio">
       <property name="text">
        <string>Передвижение точек треугольника</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>                                    Размер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Периметр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="PerLable">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Площадь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AreaLable">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Центр масс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CentrLable">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>287</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="lastFigureButton">
         <property name="text">
          <string>Прошлая фигура</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextFigureButton">
         <property name="text">
          <string>Следующая фигура</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="clearAllButton">
         <property name="text">
          <string>Очистить все</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearOneButton">
         <property name="text">
          <string>Удалить выбранное</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
